integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -104,6 +104,13 @@
                         }
                     },
                     {
+                        bidder: 'adgear',
+                        // optional params: [ 'url', 'sizes' ]

```

Comment:
Would you be able to provide examples of `url` and `sizes` ? Is there a reason to have `sizes` optionally here vs level up on the adUnit?